src/shared/GlobalTopBanners/GlobalTopBanners.spec.tsx

```diff
@@ -1,14 +1,45 @@
 import { render, screen } from '@testing-library/react'
+import { MemoryRouter, Route } from 'react-router-dom'
 
-import GlobalBanners from './GlobalTopBanners'
+import GlobalTopBanners from './GlobalTopBanners'
 
 jest.mock('./SentryTrialBanner', () => () => 'SentryTrialBanner')
+jest.mock('./RequestInstallBanner', () => () => 'RequestInstallBanner')
 
-describe('GlobalBanners', () => {
+const wrapper =
+  ({
+    initialEntries = '/bb/batman/batcave',
+    path = '/:provider/:owner/:repo',
+  }): React.FC<React.PropsWithChildren> =>
+  ({ children }) =>
+    (
+      <MemoryRouter initialEntries={[initialEntries]}>
+        <Route path={path}>{children}</Route>
+      </MemoryRouter>
+    )
```

Comment:
If you're mocking out `RequestInstallBanner` then you probably don't need this complex wrapper